logstash-core/src/main/java/org/logstash/ackedqueue/ext/JrubyAckedQueueExtLibrary.java

```diff
@@ -0,0 +1,184 @@
+package org.logstash.ackedqueue.ext;
```

Comment:
is this (and the other new file) intended to be part of this PR ?

Reply:
@jakelandis argh no rebase incorrectly :( Sorry for the noise, cleaning up ...